Amplify/Categories/Auth/Models/AuthSignInStep.swift

```diff
@@ -46,6 +46,13 @@ public enum AuthSignInStep {
     /// Auth step that required the user to be confirmed
     ///
     case confirmSignUp(AdditionalInfo?)
+    
+    /// Auth Passwordless
+    /// Confirm sign in with OTP
+    case confirmSignInWithOTP(AuthCodeDeliveryDetails, AdditionalInfo?)
```

Comment:
Yup - here is the sample response for sign in request

```
{
    "ChallengeName": "...",
    "ChallengeParameters": {
        "nextStep": "PROVIDE_CHALLENGE_RESPONSE",
        "codeDeliveryDetails": {
            "attributeName": "email",
            "deliveryMedium": "EMAIL",
            "destination": "S***@g***"
        }
    },
    "Session": "..."
}
```

This will help populate the `AuthCodeDeliveryDetails` object.